modules/sc-mesh-secure-deployment/src/2_0/features/cbma/standalone_cbma.py

```diff
@@ -0,0 +1,193 @@
+
+    logger.addHandler(console_handler)
+
+
+def shutdown_interface(interface_name):
+    subprocess.run(["ip", "link", "set", interface_name, "down"],
+                   capture_output=True)
+
+
+def destroy_interface(interface_name):
+    subprocess.run(["ip", "link", "delete", interface_name],
+                   capture_output=True)
+
+
+def delete_ebtables_rules():
+    command_ebtables = ["ebtables", "-t", "nat", "-L", "OUTPUT"]
+    command_sed = ["sed", "-n", "/OUTPUT/,/^$/{/^--/p}"]
+    command_xargs = ["xargs", "ebtables", "-t", "nat", "-D", "OUTPUT"]
+
+    proc_ebtables = subprocess.Popen(command_ebtables, stdout=subprocess.PIPE)
+    proc_sed = subprocess.Popen(
+        command_sed, stdin=proc_ebtables.stdout, stdout=subprocess.PIPE
+    )
+    proc_xargs = subprocess.Popen(command_xargs, stdin=proc_sed.stdout,
+                                  stdout=subprocess.DEVNULL,
+                                  stderr=subprocess.DEVNULL)
+    proc_xargs.wait()
+
+    subprocess.run(["ebtables", "--delete", "FORWARD",
+                    "--logical-in", "br-upper", "--jump", "ACCEPT"],
+                   capture_output=True)
+    subprocess.run(["ebtables", "--delete", "FORWARD",
+                    "--logical-out", "br-upper", "--jump", "ACCEPT"],
+                   capture_output=True)
+
+
+def delete_macsec_links():
+    cmd = "awk -F/ '/macsec/{$0=FILENAME; system(\"ip link delete \" $5)}' /sys/class/net/*/uevent"
+    subprocess.run(cmd, shell=True, capture_output=True)
+
+
+def cleanup_cbma(bat_iface: str) -> None:
+    shutdown_interface(bat_iface)
+    delete_ebtables_rules()
+    delete_macsec_links()
+    destroy_interface(bat_iface)
+
+
+def stop_cbma(bat_iface: str):
+    try:
+        logger.info("Stopping CBMA...")
+
+        for process in cbma_processes:
+            try:
+                if process.is_alive():
+                    process.terminate()
+            except Exception as e:
+                logger.error(f"Terminating process error: {e}")
+
+        for process in cbma_processes:
+            try:
+                if process.is_alive():
+                    process.join(timeout=1.0)
+                    process.kill()
+            except Exception as e:
+                logger.error(f"Killing process error: {e}")
+
+        logger.debug("CBMA processes terminated, continue cleanup...")
+        cleanup_cbma(bat_iface)
+        logger.debug("CBMA cleanup finished")
+
+    except Exception as e:
+        logger.error(f"Stop CBMA error: {e}")
+
+
+def main(interfaces: List [str],
+         port: int,
+         bat_iface: str,
+         cert_folder: str,
+         cert_chain: str) -> None:
+
+    cleanup_cbma(bat_iface)
+
+    batman(bat_iface)
+
+    for iface in interfaces:
+        try:
+            with open(f"/sys/class/net/{iface}/phy80211/index") as f:
+                index = f.read()
+                wpa_supplicant_control_path = f"/var/run/wpa_supplicant_id{index}/{iface}"
```

Comment:
Index in wpa_supplicant_id{index} is a different thing than /sys/class/net/{iface}/phy80211/index.   

wpa_supplicant_id{index} is a configuration index which is used in Comm_control based configurations.  Not mapping to phy index.